app/src/debug/google-services.json

```diff
@@ -15,6 +15,7 @@
       },
       "oauth_client": [
         {
```

Comment:
This one was just an empty line I missed while resolving merge conflicts in the web editor

Reply:
Cool @AnthonyGW